op-batcher/batcher/channel_config_provider_test.go

```diff
@@ -81,7 +81,7 @@ func TestDynamicEthChannelConfig_ChannelConfig(t *testing.T) {
 				blobBaseFee: tt.blobBaseFee,
 			}
 			dec := NewDynamicEthChannelConfig(lgr, 1*time.Second, gp, blobCfg, calldataCfg)
-			cc := dec.ChannelConfig()
+			cc := dec.ChannelConfig(false)
```

Comment:
here would then become
```suggestion
			cc := dec.ChannelConfig(tt.isPectra)
```